src/containers/Nodes/columns/hooks.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import {useSelectedColumns} from '../../../utils/hooks/useSelectedColumns';
+
+import {getNodesColumns} from './columns';
+import {
+    DEFAULT_NODES_COLUMNS,
+    NODES_COLUMNS_TITLES,
+    NODES_TABLE_SELECTED_COLUMNS_LS_KEY,
+    REQUIRED_NODES_COLUMNS,
+} from './constants';
+import type {GetNodesColumnsProps} from './types';
+
+export function useNodesColumns(params: GetNodesColumnsProps) {
+    return React.useMemo(() => {
```

Comment:
It seems this wrapper is redundant. We do may use `getNodesColumns(params)` straightforward, don't we?

Reply:
Removed `useNodesColumns` hook, but left `useMemo` - its result is used in dependencies in hooks inside `useSelectedColumn`